prow/cluster/deck_deployment.yaml

```diff
@@ -64,6 +64,18 @@ spec:
         - name: job-config
           mountPath: /etc/job-config
           readOnly: true
+        livenessProbe:
```

Comment:
Isn't this code new? If so we need to wait until its on a version that has it

Reply:
As per the tracking issue I believe it's already deployed -- could you please confirm?

Reply:
This file says what commit is deployed: gcr.io/k8s-prow/deck:v20190307-fd53164

Is your change in that?

Reply:
Oops, messed up my check -- it's barely not. Do you want to bump to grab it? You could also do both in one PR and not incur one slow 502-ridden rollout

Reply:
I'll want to bump again today once we get my tide fix and the config PR in, yes